instances/technique/psychologicalTesting.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/psychologicalTesting",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Psychological testing' is the administration of psychological tests typically by trained evaluators which also evaluate the person's responses according to carefully prescribed guidelines resulting in psychometric scores are thought to reflect individual or group differences in the construct the test purports to measure. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Psychological_testing)]",
```

Comment:
The definition is a bit long and confusing. 

Definition: "'Psychological testing' is a psychometric measurement to evaluate a person's response to a psychological test according to carefully prescribed guidelines. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Psychological_testing)]"
 
The rest could be moved into the description.

Reply:
I like it. Thanks. And I updated it. Let's do a more detailed description later.

Reply:
but crashed the code... wait a moment